.github/workflows/upload-to-pypi.yml

```diff
@@ -48,9 +48,7 @@ jobs:
       - name: Install cibuildwheel and dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install cibuildwheel
-          pip install -r requirements-dev.txt
-          pip install setuptools wheel twine
+          python -m pip install cibuildwheel setuptools wheel twine
```

Comment:
Isn't it possible to have a `build` section under `[options.extras_require]` in `setup.cfg` 
Would allow to get rid of this line and be consistent with `test`, `dev`, `display`

Reply:
Good idea.

I would add a `publish` section in `[options.extras_require]` with `cibuildwheel` and `twine`, then change the gh action accordingly.

Reply:
I will dig into this !

Reply:
@deepcharles 
Forget about it, since it is actually `cibuildwheel` that builds the package, and that we do not explicitly perform a `python -m pip install .[whatever]`, then the idea described above does not apply !

Reply:
I guess we could still replace 

```
python -m pip install cibuildwheel setuptools wheel twine
```

by

```
python -m pip install cibuildwheel twine
```

Reply:
Sure ! 
But we will not be able to use setup.cfg to rationalize package management. No big deal as of now !